components/uploadForm/form.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState, useRef } from 'react';
+import axios from 'axios';
+import { EStep } from '../../constants';
+import FileIcon from '../../static/file.png';
+import UploadIcon from '../../static/upload.png';
+
+interface IProps {
+  setStep: React.Dispatch<React.SetStateAction<EStep>>;
+  setResp: React.Dispatch<React.SetStateAction<any>>;
+}
+
+const Form: React.FC<IProps> = (props: IProps) => {
+  const fileBox = useRef(null);
+  const [fileArray, setfileArray] = useState([]);
+  const [dragging, setdragging] = useState(false);
+
+  /* form input and submit */
+  const handleInput = (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    if (fileBox.current !== null) {
+      setfileArray([fileBox.current.files[0]]);
+    }
+  };
+  const handleSubmit = async (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    const formData = new FormData();
+    formData.append('file', fileArray[0]);
+
+    // tslint:disable
+    const d = await axios
+      .post(`${process.env.MAIN_HOST}/students/validate/`, formData, {
+        headers: new Headers({ 'Content-Type': 'multipart/form-data"' }),
+      })
+      .then(resp => resp.data)
+      .catch(err => {
+        console.log(err);
+        return null;
+      });
+    // tslint:enable
+    if (d.appled && d.error_msg === 'success') {
```

Comment:
`applied` ??

Reply:
It is from api docs.

Reply:
I have updated in [new commit](https://github.com/wiasliaw77210/digital-certificate-edu/pull/1/commits/adba9f8438c97a8803c273d99e2ab252dfd7dce9).

And change it from api res~